src/main.py

```diff
@@ -0,0 +1,8 @@
+# Main entry point to run the recommendation pipeline
+
+from data_processing.wiki_to_netflix import * 
+
+if __name__ == '__main__':
+    test = len(sys.argv) < 2 or sys.argv[1] != '--prod'
```

Comment:
nit: Looks like we lost the comment here, can we try to re-add it?

Reply:
I think this is the code that was lost which I added back:

q = wiki_to_netflix.format_sparql_query('The Room', 2003)
print(q)

Reply:
Sorry I meant this comment:

```
# Test is true if no argument is passed or if the first argument is not '--prod'.
```

But now I see that the comment is still in the `wiki_to_netflix.py` script, it just didn't get copied over when the check for `--prod` got copied. No big deal.